unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/WorldChatWindowController.cs

```diff
@@ -592,6 +599,31 @@ private void HandleChannelOpened(string channelId, string previousChannelId)
         OpenPublicChat(channelId);
     }
 
+    private void HandleAskForJoinChannel(string channelName)
+    {
+        chatController.OnAskForJoinChannel -= HandleAskForJoinChannel;
+
+        if (!channelsFeatureFlagService.IsAllowedToCreateChannels())
+            return;
+
+        if (rendererState.Get())
+            dataStore.channels.currentJoinChannelModal.Set(channelName, true);
+        else
+        {
+            channelToJoinAtTheBeginning = channelName;
+            rendererState.OnChange += HandleAskForJoinChannelAfterRendererState;
+        }
+    }
+
+    private void HandleAskForJoinChannelAfterRendererState(bool current, bool previous)
```

Comment:
[cosmetic]: you can rename `previous` to `_`, since it is not used

Reply:
Done!